sqlparser/sql.y

```diff
@@ -550,6 +551,14 @@ isolation_level:
     $$ = &SetExpr{Name: NewColIdent("tx_isolation"), Expr: NewStrVal([]byte("serializable"))}
   }
 
+set_operation_scope:
+  set_session_or_global
+  | LOCAL
```

Comment:
can we mention that LOCAL is postgresql specific? 
https://dev.mysql.com/doc/refman/5.7/en/set-variable.html - in mysql's docs there is nothing about local
https://www.postgresql.org/docs/current/sql-set.html - but supported by postgres